src/datasets/features/DatasetVersionFileColumns/DatasetVersionFileColumns.tsx

```diff
@@ -77,6 +77,13 @@ const DatasetVersionFileColumns = (props: DatasetVersionFileColumnsProps) => {
     };
   }, [data]);
 
+  const renderAttributePercentage = useCallback((value: number) => {
```

Comment:
Do you need this ? You have the `total` set above so you could directly use that in the render

Reply:
It was more to separate the way we render the stats (in case we want to change it in the future), but not really needed you're right.